doc/webui/README.md

```diff
@@ -117,10 +111,9 @@ API to allow instances to register themselves with `clusterd`
 
 + Request (application/json)
 
-        {
-            "name": "auto-registered instance",
-            "host": "192.168.0.8"
-        }
+    + Attributes (Base Instance)
+        + host: 192.168.0.8
+        + name: autoregistered instance
```

Comment:
If you specify the same fields here and in the *Base Instance* model again, you'll override the *Base Instance* model with these values. If you don't want to override anything, you should not specify them twice I think.